org/contributors.yml

```diff
@@ -1,656 +1,207 @@
-- fredwangwang
-- friegger
-- frodenas
-- gab-satchi
-- galenhammond
-- gaowa0215
-- GarfieldLinux
-- GarimaSharma
-- garrying
-- gbandres98
 - gcapizzi
-- gdankov
 - geofffranks
 - georgethebeatle
-- georgi-lozev
 - Gerg
-- gertd
 - gggevorgyan
-- git-narya
-- gmrodgers
-- gossion
-- GowriRegistry
-- graeme-davison
-- grahamlutz
-- gsiener
-- gu-bin
 - gururajsh
-- harshah
-- haydonryan
-- hcf-bot
-- HeavyWombat
-- heidmotron
-- hemarsai
-- herrjulz
-- heyjcollins
 - hmanukyanVMw
-- hnandiwada
-- hoegaarden
 - hsinn0
-- hyakuhei
 - iaftab-alam
-- idoru
 - ifindlay-cci
-- IgorBelyi
-- ihcsim
 - ikasarov
-- ilackarms
-- iplay88keys
 - IvanBorislavovDimitrov
 - ivanhristov98
-- jackfengibm
-- jacksoncvm
-- JacquesPerrault
-- james-masson
-- jamieklassen
-- jamiemonserrate
-- jandubois 
 - jaristiz
 - jbooherl
-- jchenry
-- jdeppe-pivotal
 - jdgonzaleza
-- jeff-hobbs
-- jerrybelmonte
-- jessepye
-- jgawor
-- jhasu
 - jhvhs
-- jianqiu
-- jimbo459
 - jimconner
-- jimmykarily
 - jintusebastian
-- jkjell
-- jkonicki
-- jmarrero
-- jncd
 - jochenehret
-- joeeltgroth
 - joefitzgerald
-- joeldsa
 - joergdw
 - johha
-- johncornish
-- JordanICollier
-- josephgee
-- joshlong
-- joshuatcasey
 - jpalermo
-- jpatel-pivotal
-- jpmcb
 - jrussett
-- jsampson1
 - julian-hj
-- Justin-W
-- k1tm
-- KaiHofstetter
-- kaleo211
-- kartiklunkad26
 - kathap
-- kauana
-- KauzClay
-- kcboyle
-- KeepAustinWired
-- kejadlen
 - KevinJCross
 - Kiemes
 - kieron-dev
 - kimago
-- kirederik
 - klakin-pivotal
 - klapkov
-- KlapTrap
 - kmarkwardt-vmware
-- knm3000
 - kohara88
 - korifi-bot
-- kreinecke
-- kriscott
-- krook
-- ktchen14
-- ktollivervmw
-- ktpv
-- kushmerick
-- langered
-- lbarncord
 - leeporte
-- lenapivotal
-- lisamburns
-- LizClaireMorris
 - lnguyen
 - loewenstein
-- loganloganlogan
-- lterminasyan
-- lwoydziak
-- maheshkumarvs
-- malba
-- malik-altaf
-- malston
-- mamachanko
-- Manifaust
-- manno
 - MarcPaquette
 - mariash
-- markstokan
 - markthemarkest
-- Markus-Ehrnsperger
-- martyspiewak
 - matt-royal
-- mattcui
-- mattmcneeney
-- mauromorales
 - max-soe
-- maximilien
-- mcwumbly
-- menicosia
 - MerricdeLauney
-- metatype
 - metric-store-ci
-- mfine30
-- mgibson1121
-- michaelklishin
-- miguelverissimo
-- mihaibuzgau
 - MirahImage
-- mjgutermuth
-- mjj209
-- mkjelland
-- mklanjsek
 - mkocher
-- mkuratczyk
-- MMisoch
-- MoatazBM
-- modelseer
 - moggibear
 - moleske
-- monadic
-- monamohebbi
-- mook-as
-- mrdavidlaing
 - mrosecrance
-- msmykowski
 - mvach
-- mvalliath
 - myeghiazarya
-- N00BEST
-- n4wei
-- nader-ziada
 - nand2
-- natalieparellano
-- navdeep-pama
 - nebhale
-- nedenwalker
-- neil-hickey
-- nguerette
-- nhsieh
-- nickjameswebb
-- nickstreet
-- nicregez
-- nierajsingh
-- nimakaviani
-- Nino-K
-- nkguy
-- nmaslarski
 - notrepo05
 - nouseforaname
 - nvvalchev
-- nwmac
-- objectfox
-- ograves-pivotal
 - ohkyle
-- ojhughes
-- omerbensaadon
-- oppegard
-- OpusDude
-- ozbasarir
 - pabloarodas
-- pabloschuhmacher
-- paigecalvert
-- paltanmoy
-- paulcwarren
```

Comment:
@paulcwarren in execution lead of the Vulnerability Management WG and should probably not be removed.

[Vulnerability Management WG](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/vulnerability-management.md) is missing the yaml section in the WG charter and therefore users with a WG role are not excluded from inactive users.

In addition, all users with a WG role will get added to the cloundfoundry org even if not listed explicitly in contributors.yml. See also [org management readme](https://github.com/cloudfoundry/community/blob/main/org/readme.md#organization-members).

Reply:
I started a [discussion](https://cloudfoundry.slack.com/archives/C026XJGNC2U/p1703061747301259?thread_ts=1703061394.243059&cid=C026XJGNC2U) about this in our toc slack channel as discussed during the TOC meeting on 19th of December.

Reply:
This has been addressed with https://github.com/cloudfoundry/community/pull/746 and the latest run doesn't include @paulcwarren any more. I'm going to run the automation with user notification activated.